ssh_check/datadog_checks/ssh_check/ssh_check.py

```diff
@@ -1,13 +1,27 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+import re
 import time
 from collections import namedtuple
 
 import paramiko
 
 from datadog_checks.checks import AgentCheck
 
+# Example ssh remote version: http://supervisord.org/changes.html
+#   - SSH-2.0-OpenSSH_8.1
+SSH_REMOTE_VERSION_PATTERN = re.compile(
+    r"""
+    ^.*_
+    (?P<major>0|[1-9]\d*)
+    \.
+    (?P<minor>0|[1-9]\d*)
+    (?P<release>0|p[1-9]\d*)?
```

Comment:
```suggestion
    (?P<release>p0|p[1-9]\d*)?
```

or this, if p0 is not a valid case

```suggestion
    (?P<release>p[1-9]\d*)?
```